packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputElement.tsx

```diff
@@ -133,113 +40,185 @@ const getFingerprint = (
-                    <OutputRightLine data-testid={`@modal/output-${line.id}`} key={line.id}>
-                        <OutputHeadline data-testid="@modal/output-headline">
-                            {isActive && (line.id === 'address' || line.id === 'regular_legacy')
-                                ? line.confirmLabel
-                                : line.label}
-                        </OutputHeadline>
-                        <OutputValue data-testid="@modal/output-value">
-                            {isActive &&
-                            displayMode &&
-                            TYPES_TO_BE_DISPLAYED_IN_SCREEN_BOX.includes(line.id) ? (
-                                <DeviceDisplay displayMode={displayMode} address={line.value} />
-                            ) : (
-                                <OutputValueWrapper>
-                                    {line.plainValue ? (
-                                        line.value
-                                    ) : (
-                                        <FormattedCryptoAmount
-                                            disableHiddenPlaceholder
-                                            value={line.value}
-                                            symbol={
-                                                // TX fee is so far always paid in network native coin
-                                                line.id !== 'fee' && token ? token.symbol : symbol
-                                            }
-                                            contractAddress={token?.contract}
-                                        />
-                                    )}
-                                </OutputValueWrapper>
-                            )}
-                            {/* temporary solution until fiat value for ERC20 tokens will be fixed  */}
-                            {symbol && fiatVisible && !(line.id !== 'fee' && token) && (
-                                <>
-                                    <DotSeparatorWrapper>
-                                        <DotSeparator />
-                                    </DotSeparatorWrapper>
-                                    <OutputValueWrapper>
-                                        <FiatValue
-                                            disableHiddenPlaceholder
-                                            amount={line.value}
-                                            symbol={symbol}
-                                        />
-                                    </OutputValueWrapper>
-                                </>
+        <Card paddingType="none" fillType={state === 'done' ? 'flat' : 'default'}>
+            <Row padding={{ vertical: spacings.sm, horizontal: spacings.md }} gap={spacings.sm}>
+                <StatusIndicator state={state} />
+                <H4 typographyStyle={state !== 'pending' ? 'callout' : 'hint'}>{title}</H4>
+            </Row>
+            <Divider margin={{}} />
```

Comment:
Margin is not necessary

Reply:
It is necessary to do it like this, otherwise it would apply the default margin.

Reply:
I see it now 👍 OK... 😅